docs/guide/setup-examples/julia.md

```diff
@@ -0,0 +1,401 @@
+place the cursor on a line and type <Key k="ctrl+enter" />.
+
+#### Running Julia files
+
+In a opened Julia source file, press <Key k="ctrl+b" />
+(which is the key binding for **Tools → Build**).
+The current file will be evaluated in a Julia-running Terminus window
+with an `include()`.
+
+### Asking for help in the REPL
+
+From a Julia source file, press <Key k="ctrl+shift+x, ctrl+h"> to ask the Julia
+REPL to display help about the current selection.
+
+## Zeal
+
+Zeal is an open-source browser of off-line documentation.
+There is a Sublime package that allows you to jump to Zeal documentation
+for the symbol under cursor.
+Zeal can also be used with other languages than Julia.
+Zeal can be downloaded from [Zeal website][].
+The documentation for Julia needs to be downloaded within the Zeal documentation browser:
+in the menu "Tools" choose "Docsets".
+
+From Package Control install [Zeal for Sublime Text][].
+
+The Zeal executable needs to be revealed to the editor.
+Also, Zeal
+needs to be made aware of the language of the documentation request:
+
+```json
+{
+  /**
+   Zeal executable path.
+   The instructions below require zeal
+   to be located very precisely by its full path!?
+   */
+  "zeal_command": "C:\\Program Files\\Zeal\\zeal.exe",
+  "mapping_sort" : true,
+
+  // Language mapping.
+  "language_mapping": {
+    "Python": {"lang": "python", "zeal_lang": "python"},
+    "Julia": {"lang": "julia", "zeal_lang": "julia"}
+  }
+}
+```
+
+Zeal can be brought up by pressing `F1` or through the Command Palette.
+
+[Zeal website]: https://zealdocs.org/download.html
+[Zeal for Sublime Text]: https://packagecontrol.io/packages/Zeal
+
+## Starts Julia from Git Bash 
+
+I really can't stand the default "shell" (CMD)
+in which Julia starts on Windows.
+The interaction with the shell is quite unsatisfactory then:
+most of the Windows commands and all of the UNIX commands
+don't work as one would expect.
+What I prefer instead is [Git Bash](https://git-scm.com/downloads).
+One can get that to run Julia by starting Sublime Text  from a bat file.
+I create such a file with the line:
+
+```
+cmd /C
+start "" "%PROGRAMFILES%\\Git\\bin\\sh.exe" --login -i -c "exec
+\"C:\Users\PetrKrysl\Documents\Productivity\PortableSublimeText\sublime_text.exe\""
+```
+
+where my portable Sublime Text executable is invoked from within the Git shell.
+
+In order to get an executable file with the Sublime Text icon,
+I create a shortcut (for instance to be placed on the desktop),
+and I give it the [ST icon][].
+
+Alternatively, start the Git Bash terminal,
+and run the `subl` executable of your portable Sublime Text.
+When a terminal is opened inside the running editor, it runs the Git Bash.
+In particular, when Julia is started in the editor,
+its shell mode drops you to the Bash. Perfect!
+
+[ST icon]: https://github.com/PetrKryslUCSD/HowToUseJuliaWithSublimeText3/blob/master/sublime-text.ico
+
+## Credits
+
+Thanks to Petr Krysl, Paul Soderlind, @mbauman.
```

Comment:
`@mbauman` doesn't really help without a namespace or a URL. Is this a handle on Twitter, Github or some other random site?